server/routes/userRoutes.ts

```diff
@@ -19,5 +20,5 @@ router.delete('/interview/:id', userAuth, deleteInterview);
 router.get('/user/review', userAuth, getAllReviews);
 router.get('/user/info/:id', getUserById);
 router.get('/user/review', userAuth, getAllReviews);
-
+router.get('/users/interviewer/available', userAuth, getInterviewerAvailableTime);
```

Comment:
This request is only for the interviewer.

Reply:
I will use the middleware from the create review branch when it's merged